src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -435,14 +429,90 @@ public DcSensitivityAnalysis(MatrixFactory matrixFactory) {
         super(matrixFactory);
     }
 
-    protected void setFunctionReferenceOnFactors(List<LfNetwork> lfNetworks, List<LfSensitivityFactor> factors,
-                                                 LoadFlowParameters lfParameters, OpenLoadFlowParameters lfParametersExt) {
-        DcLoadFlowParameters dcLoadFlowParameters = new DcLoadFlowParameters(lfParametersExt.getSlackBusSelector(), matrixFactory,
-                true, lfParametersExt.isDcUseTransformerRatio(), lfParameters.isDistributedSlack(),  lfParameters.getBalanceType());
-        DcLoadFlowResult dcLoadFlowResult = new DcLoadFlowEngine(lfNetworks, dcLoadFlowParameters).run();
+    private GraphDecrementalConnectivity<LfBus> createConnectivity(LfNetwork lfNetwork) {
```

Comment:
@floriand-e2r Can you move this function to a network util class ?

Reply:
I've put it in LfNetwork directly (similarly to `createZeroImpedanceSubGraph()`)